package.json

```diff
@@ -115,6 +115,7 @@
     "coffee-loader": "0.7.3",
     "coffee-script": "1.12.7",
     "constants": "0.0.2",
+    "cw-components": "github:ClimateWatch-Vizzuality/climate-watch-components#1.51.0",
```

Comment:
If we are using a custom sankey do we need this lib?

Reply:
You're right, removed 👍